### PR TITLE
Adding Nav and Filter toggle

### PR DESCRIPTION
**Pull request by jwildfire**
Documented in #93

### CODE REVIEW

**Comment by jwildfire** [comment]
Also added in the new nav layout described in #105.

File: src/codebook/controls/filters/init.js [hunk 1]
```diff
@@ -24,18 +24,17 @@ export function init(codebook) {
...
-
+  var filterList = selector.append("ul").attr("class", "filter-list");
   var filterItem = filterList
     .selectAll("li")
     .data(codebook.config.filters)
     .enter()
     .append("li")
     .attr("class", function(d) {
-      return "custom-" + d.key + " filterCustom";
+      return "custom-" + d.value_col + " filterCustom";
```
Comment (samussiah) [inline src/codebook/controls/filters/init.js]: Does this affect how the user specifies the filters object, i.e. `{key:` vs `{value_col:`?

Comment (jwildfire) [inline src/codebook/controls/filters/init.js]: It's currently `value_col`, the class was just wrong - it was rendering as `custom-undefined` for all levels. This just updates the class to be accurate

**Comment by jwildfire** [comment]
Merging. Made issue #109 to cover visibility of the show/hide chart buttons.

**Pull request merged**